app/models/PeriodBreakdown.scala

```diff
@@ -110,18 +112,89 @@ sealed trait PhaseTwoPeriodBreakdown {
 }
 
 final case class PhaseTwoFurloughBreakdown(grant: Amount, paymentWithPeriod: PaymentWithPhaseTwoPeriod, furloughCap: FurloughCap)
-    extends PhaseTwoPeriodBreakdown
+    extends PhaseTwoPeriodBreakdown {
+  def isCapped: Boolean = (paymentWithPeriod.referencePay.value * 0.8) > furloughCap.value
+  def calculatedFurlough: String = Amount(paymentWithPeriod.referencePay.value * 0.8).halfUp.value.formatted("%.2f")
+}
 
 final case class PhaseTwoNicBreakdown(
   grant: Amount,
   paymentWithPeriod: PaymentWithPhaseTwoPeriod,
   threshold: Threshold,
   nicCategory: NicCategory)
-    extends PhaseTwoPeriodBreakdown
+    extends PhaseTwoPeriodBreakdown {
+  def isPartial = paymentWithPeriod.phaseTwoPeriod.periodWithPaymentDate.period.isInstanceOf[PartialPeriod]
+  def isPartTime = paymentWithPeriod.phaseTwoPeriod.isPartTime
+
+  def thresholdMessage(implicit messages: Messages): String =
+    (isPartial, isPartTime) match {
+      case (false, false) => messages("phaseTwoNicBreakdown.l3", threshold.value.formatted("%.2f"))
+      case (true, false) =>
+        messages(
+          "phaseTwoNicBreakdown.l3.partial",
+          threshold.value.formatted("%.2f"),
```

Comment:
minor :  we can have an implicit or something  - with out using `.formatted("%.2f")` explicitly several times  -  but not a big deal we can leave it for now